scripts/secure-boot/generate-images/generate-image.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
```

Comment:
It seems we would be fine with just: `scripts/secure-boot` path

Reply:
Fixed https://github.com/Dasharo/open-source-firmware-validation/pull/187/commits/13e2018fef5b3b1f047a06eb4538705a8f4c1141